src/main/resources/locales/en/subscription-urn-search.json

```diff
@@ -1,6 +1,6 @@
 {
   "title": "Subscription URN search",
-  "header": "Enter a unique reference number",
+  "header": "What is the unique reference number (URN)?",
```

Comment:
It looks like it was right the first time based on the screenshots from Nige

Reply:
I have taken the text from the herokuapp portal

Reply:
we should be using the screenshots on the ticket unless stated otherwise to take from heroku, heroku is used for prototyping and therefore will be different in some content as different things will be being trialled, so as default we should be taking the approved screenshots from the ticket

Reply:
unresolving as i think you must have accidentally resolved